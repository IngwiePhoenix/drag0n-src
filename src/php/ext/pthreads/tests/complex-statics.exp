sql::_query: SELECT * FROM mysql.user
sql::_query: SELECT * FROM mysql.user
sql::_query: SELECT * FROM mysql.user
sql::_query: SELECT * FROM mysql.user