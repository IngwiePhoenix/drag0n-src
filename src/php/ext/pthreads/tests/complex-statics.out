sql::_query: SELECT * FROM mysql.user
sql::_query: SELECT * FROM mysql.user

Warning: Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) in Unknown on line 0
sql::_query: SELECT * FROM mysql.user
sql::_query: SELECT * FROM mysql.user